homeassistant/components/bring/sensor.py

```diff
@@ -0,0 +1,117 @@
+"""Sensor platform for the Bring! integration."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from enum import StrEnum
+
+from bring_api import BringUserSettingsResponse
+from bring_api.const import BRING_SUPPORTED_LOCALES
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import BringConfigEntry
+from .coordinator import BringData, BringDataUpdateCoordinator
+from .entity import BringBaseEntity
+from .util import list_language, sum_attributes
+
+
+@dataclass(kw_only=True, frozen=True)
+class BringSensorEntityDescription(SensorEntityDescription):
+    """Bring Sensor Description."""
+
+    value_fn: Callable[[BringData, BringUserSettingsResponse], StateType]
+
+
+class BringSensor(StrEnum):
+    """Bring sensors."""
+
+    URGENT = "urgent"
+    CONVENIENT = "convenient"
+    DISCOUNTED = "discounted"
+    LIST_LANGUAGE = "list_language"
+
+
+SENSOR_DESCRIPTIONS: tuple[BringSensorEntityDescription, ...] = (
+    BringSensorEntityDescription(
+        key=BringSensor.URGENT,
+        translation_key=BringSensor.URGENT,
+        value_fn=lambda lst, _: sum_attributes(lst, "urgent"),
+    ),
+    BringSensorEntityDescription(
+        key=BringSensor.CONVENIENT,
+        translation_key=BringSensor.CONVENIENT,
+        value_fn=lambda lst, _: sum_attributes(lst, "convenient"),
+    ),
+    BringSensorEntityDescription(
+        key=BringSensor.DISCOUNTED,
+        translation_key=BringSensor.DISCOUNTED,
+        value_fn=lambda lst, _: sum_attributes(lst, "discounted"),
+    ),
```

Comment:
let's add unit of measurements

Reply:
Ok, will do, wasn't sure about that. I hope some day we can have translatable units ^^